docs/userguides/legalhold.md

```diff
@@ -0,0 +1,96 @@
+# Manage legal hold custodians
+
+Once you [create a legal hold matter in the Code42 console](https://support.code42.com/Administrator/Cloud/Configuring/Create_a_legal_hold_matter#Step_1:_Create_a_matter), you can use the Code42 CLI to add or release custodians from the matter.
+
+Use the `legal-hold` commands to manage legal hold custodians.
+ - To see a list of all the users currently in your organization, you can export a list from the Users action menu.
```

Comment:
Is this reference in any support docs?

Reply:
ah yep, that was in the support CLI docs that I pulled these lines from, but I missed the link.